openwisp_controller/config/static/import_export/import.css

```diff
@@ -0,0 +1,81 @@
+.import-preview .errors {
+  position: relative;
+}
+
+.validation-error-count {
+  display: inline-block;
```

Comment:
the same applies here and on any other propert which does not relate to colors.
Duplicate code is an enemy because it makes maintenance harder.